django/projects/mysite/settings.py.example

```diff
@@ -93,3 +93,5 @@ IMPORTER_DEFAULT_IMAGE_BASE = 'http://CATMAID_SERVERNAME/CATMAID_SUBDIR/data'
 STATIC_EXTENSION_URL = '/CATMAID_SUBDIR/staticext/'
 # The absolute local path where the static extension files live
 STATIC_EXTENSION_ROOT =  'CATMAIDPATH/django/staticext'
+
+INSTALLED_APPS += tuple('{}.apps.{}Config'.format(app_name, app_name.title()) for app_name in INSTALLED_EXTENSIONS)
```

Comment:
Would you mind adding a short comment above this line explaining that this makes installed extensions available? The `settings.py` file might get edited manually and for this it would be useful to have a short explanation what this line does.

Reply:
Will do, good point.